data/json/proficiencies/chemistry.json

```
@@ -1,4 +1,24 @@
 [
+  {
+    "type": "proficiency",
+    "id": "prof_intro_chemical",
+    "name": { "str": "Principles of Chemical Production" },
+    "description": "You're beginning to grasp the basics of chemical production.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_fail_multiplier": 1.5,
```

```suggestion
    "default_fail_multiplier": 5,
```
As a general rule, any "intro to" proficiency should have a high fail multiplier. If you lack even the basics, it's really hard to do anything.